### PR TITLE
Ft manage request

### PR DESCRIPTION
**Pull request by Noel2002**
# Description
This branch was created to allow managers view categorised travel requests from requestors and be able to approve or reject them.

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

- Clone this repo
- Run the command `npm i`
- Run the command `npm start`

# Checklist:

- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

### CODE REVIEW

**Comment by salviosage** [comment]
Please rebase

**Comment by salviosage** [comment]
please rebase

**Review by songa1 — COMMENTED** [review]
Everything is working. Except that I'm getting `Invalid grant` error which I think is coming from Notifications or Email.

**Review by songa1 — COMMENTED** [review]
I think that will be solved after someone fix it.

**Review by edgar929 — DISMISSED** [review]

**Review by Blaiseniyo — COMMENTED** [review]
For almost everything is ok, one feedback I would give is that when you are going to approve or reject a travel request there is a window that opens. I would suggest the after approving or rejecting a request is successful the window could close itself without waiting for the user to click outside it to close it. and I think when you do that everything be will ok.

**Comment by Anguandia** [comment]
Please check on why CI is failing too

**Review by Cyusa144 — DISMISSED** [review]

File: src/components/manageTravel/ErrorModal.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+//         onClose={handleClose}
+//         aria-labelledby="simple-modal-title"
+//         aria-describedby="simple-modal-description"
+//       >
+//         {body}
+//       </Modal>
+//     </div>
+//   );
+// }
+// export default ErrorModal;
```
Comment (Anguandia) [inline src/components/manageTravel/ErrorModal.js]: Please clean this up

**Review by Anguandia — DISMISSED** [review]
Good work done. Please take note of the following for future PRs:
- The PR is too bulky, could have been broken into several smaller tasks, work breakdown wasn't adequate (52 files with a total of thousands of lines to review is not appealing)
- We will need a session on refactoring at some other time

**Review by Anguandia — APPROVED** [review]

**Pull request merged**